finance/grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/AddressDao.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.typedb.examples.fraud.db;
+
+import com.typedb.examples.fraud.model.Address;
+import java.util.Hashtable;
+
+public class AddressDao {
```

Comment:
Could we implement `Dao` for the rest of our Dao classes? From an architectural standpoint, having one half of these *Dao classes implement Dao, but the other not makes this quite confusing.

Reply:
As we don't use them, we are just using them as "storage" for now. The example is scoped toward the once that are impremented.